### PR TITLE
calculateOverlappingStrategyPrices to return full accuracy

### PR DESCRIPTION
**Pull request by zavelevsky**
Before - calculateOverlappingStrategyPrices was trimming prices to their decimals limit - but now max accuracy remains

### CODE REVIEW

File: src/strategy-management/Toolkit.ts [hunk 1]
```diff
@@ -8,7 +8,6 @@ import {
   tenPow,
   formatUnits,
   parseUnits,
-  trimDecimal,
```
Comment (barakman) [inline src/strategy-management/Toolkit.ts:11]: If this function is no longer used **anywhere**, then you may also remove its implementation from the codebase.

**Review by barakman — CHANGES_REQUESTED** [review]

File: src/strategy-management/Toolkit.ts [hunk 2]
```diff
@@ -755,22 +754,12 @@ export class Toolkit {
...
-      buyPriceHigh: trimDecimal(prices.buyPriceHigh, quoteDecimals),
-      buyPriceMarginal: trimDecimal(prices.buyPriceMarginal, quoteDecimals),
-      sellPriceLow: trimDecimal(prices.sellPriceLow, quoteDecimals),
-      sellPriceHigh: trimDecimal(sellPriceHigh, quoteDecimals),
-      sellPriceMarginal: trimDecimal(prices.sellPriceMarginal, quoteDecimals),
-      marketPrice: trimDecimal(marketPrice, quoteDecimals),
-    };
-
     logger.debug('calculateOverlappingStrategyPrices info:', {
       quoteDecimals,
```
Comment (GrandSchtroumpf) [inline src/strategy-management/Toolkit.ts:758]: Looks like you don't need quoteDecimals anymore. You can remove it and make the function sync.

File: src/strategy-management/Toolkit.ts (hunk 1 above)
Comment (zavelevsky) [reply src/strategy-management/Toolkit.ts:11]: it is used in one more place

File: src/strategy-management/Toolkit.ts (hunk 2 above)
Comment (zavelevsky) [reply src/strategy-management/Toolkit.ts:758]: good catch - but note that the method signature will change

**Pull request merged**